torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -931,26 +931,57 @@ class OutlierDetector(DetectorBase):
         if p_r is above some threshold, then we consider the activations to have significant outliers
 
     Args:
-        ratio_threshold (float): The threshold for p_r to determine if there are outliers in activations
-            Should be between 0 and 1 (both non-inclusive)
+        ratio_threshold (float, optional): The threshold for p_r to determine if there are outliers in activations
+            Default: 3.5
         reference_percentile (float, optional): The denominator to find the relative scale of the 100th percentile
+            Should be between 0 and 1
+            Default: 0.975
+        ch_axis (int, optional): The channel axis being observed to determine input weight equalization
+            Default: 1
 
     * :attr:`ratio_threshold`: The threshold for p_r to determine if there are outliers in activations
-        Should be between 0 and 1
 
     * :attr:`reference_percentile`: The denominator of the top fraction to find the relative scale of the 100th percentile
+        Should be between 0 and 1
+
+    * :attr:`ch_axis`: The channel axis being observed to determine outliers
 
+    * :attr:`DEFAULT_PRE_OBSERVER_NAME`: The name of the pre-observer to be inserted for this detector
     """
 
-    def __init__(self, ratio_threshold: float = 10.0, reference_percentile: float = 0.95):
+    # names for the pre observers that are inserted
+    DEFAULT_PRE_OBSERVER_NAME: str = "model_report_pre_observer"
+
+    def __init__(self, ratio_threshold: float = 3.5, reference_percentile: float = 0.975, ch_axis: int = 1):
         # initialize the variables of interest
         self.ratio_threshold = ratio_threshold
         self.reference_percentile = reference_percentile
+        self.ch_axis = ch_axis
 
     def get_detector_name(self) -> str:
         r"""Returns the name of this detector"""
         return "outlier_detector"
 
+    def _is_supported(self, module: nn.Module, insert: bool = False) -> bool:
+        r"""Returns whether the given module is supported for observers
+
+        Any module that doesn't have children and isn't an observer itself is supported
+
+        Args
+            module: The module to check and ensure is supported
+            insert: True if this is check for observer insertion, false if for report gen
```

Comment:
this boolean flag feels a bit weird, maybe worth considering just have two `is_supported` method, one for observer insertion, one for report gen. I'm not sure how these two things are related